src/tests/test11.vtc

```diff
@@ -15,8 +15,8 @@ varnish v1 -vcl+backend {
 			curl.set_timeout(200);
 			curl.set_connect_timeout(200);
 		} else {
-			curl.set_timeout(1300);
-			curl.set_connect_timeout(1300);
+			curl.set_timeout(1310);
```

Comment:
I have a hard time believing that 10ms make a clean difference in general. Is there a known reason for that?

Reply:
The reason why I changed the timeout was that the test case reported a run time of 1.299something seconds. This seems a bit odd, but I did not consider it too relevant, have not looked at what libcurl does and just applied this change to make the test case pass.